src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md

```diff
@@ -226,8 +226,8 @@ If you need to increase this limit, contact our support team.
 Additionally, you can change the concurrency limit policy that defines how k6 Cloud acts when the organization reaches the limit and a new test run is triggered.
 Two options are available:
 
-- `Abort test`: the new test run will be automatically aborted. This is the default option.
-- `Queue test`: the new test run will be queued for execution and started once a slot is opened. A queued test will timeout if no slot is available in 6 hours.
+- `Queue test`: the new test run will be queued for execution and started once a slot is opened. A queued test will timeout if no slot is available in 6 hours. This is the default option.
```

Comment:
Question: we are changing the default policy, but only for newly created organizations (from today onwards). We preserve the previous default for the existing organizations not to confuse them (they might rely on the old default policy). Should we also mention it here in the docs, or is this to technical of a detail?

Reply:
Yes, absolutely. We are just not switching them automatically.

Reply:
I think we should create an admonition in this bullet. Here, I've written it, but do you mind filling in the values:

```
<Blockquote mod="note" title="">

On <DATE>, the default timeout changed from 6 hours to <new_value>.
To avoid confusion, k6 kept the old default for organizations who joined before the change.

If your organization uses the old default and wants to upgrade, change these settings in <location_of_resource>. 

</Blockquote>
```

Reply:
@marcinzaremba correct me if I am wrong, but we only change which status will be the current default policy - the behaviour of both statuses are the same. 

If that is the case, I think we can keep it simple and just add to your changes:  `This is the current default policy`.

Reply:
So Pepe's feedback has been included. I think what Matt suggested is also a good addition.
I can imagine a following scenario (not sure how likely it is):
If I were an existing customer just reading the docs, and I were interested in `the concurrency policy`, I could have just assumed, that I do not have to do anything, because the `queue` policy is already set (the default).
I proceed with my testing and my test is aborted. I am surprised. I go to the settings and I see that my policy is `Abort`. Why? I think the docs might be wrong.

Reply:
I've added the suggested note. @MattDodsonEnglish Can you please take a look?